### PR TITLE
Vj mod cog

### PR DESCRIPTION
**Pull request by TitanVJ**
# Adds mod only commands:
## embed(alias: em)
* allows mods to post a message in embed form
* if # of args is odd the first arg is put to the description and the rest as title content fields
* if # of args is even the no description and only fields

## modspeak(alias: warn)
* posts a warning

### CODE REVIEW

**Comment by TitanVJ** [comment]
oh ya, the invoking command message get deleted.

File: commands_to_load/Mod.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    def __init__(self, bot):
+        self.bot = bot
+    
+    @commands.command(aliases=['em'])
+    async def embed(self, ctx, *arg):
+        logger.info('[Mod embed()] embed function detected by user ' + str(ctx.message.author))
+        await ctx.message.delete()
+        logger.info('[Mod embed()] invoking message deleted')
+        
+        if not self.isMinion(ctx):
```
[the remark this answers is not in the document]
Comment (TitanVJ) [reply commands_to_load/Mod.py]: done

**Comment by modernNeo** [comment]
since this is the "Mod PR", maybe get them to also fill this out?
https://github.com/CSSS/wall_e/issues/46

**Comment by TitanVJ** [comment]
left them out of help.json on purpose because there are so few mod commands there'll be a pinned post on their usage in #council.

**Comment by modernNeo** [comment]
@TitanVJ  its still needed by the Bot_managers if we need to unload or reload the cogs though and therefore have to remember the class that they are in

**Comment by TitanVJ** [comment]
ok. does this entail modifying the help command? or is it just entries in the json file?

**Comment by modernNeo** [comment]
@TitanVJ if the Minions dont need it and only Bot_managers do, then just the help.json needs to be modified

**Comment by TitanVJ** [comment]
ill add the entries but ill also make a separate help command for minions that dm them. this one wont disappear so they can look back on it whenever

**Comment by TitanVJ** [comment]
ill add the help command tmrw.

**Comment by modernNeo** [comment]
> ill add the entries but ill also make a separate help command for minions that dm them. this one wont disappear so they can look back on it whenever

"I'll make a separate help command for minions that dm them"....dm who exactly? @TitanVJ

**Comment by TitanVJ** [comment]
future proofing since more mod commands might be made(prob will tbh)

**Comment by TitanVJ** [comment]
dm minion who called command

**Comment by modernNeo** [comment]
So, if I understand correctly. 
there will be a help command that Bot_mangers call that list the mod commands among all the other commands in the channel that the command was entered 
as well as a help command that the mods call that will DM the mod commands to the mod that called the command? @TitanVJ

**Comment by TitanVJ** [comment]
i've realized the redundancy of this so im modifying help command to include the mod commands for bot managers and minions

**Comment by TitanVJ** [comment]
@modernNeo done. that should be everything for this one.

**Comment by modernNeo** [comment]
since this is the "Mod PR", maybe get them to also fill this out?
https://github.com/CSSS/wall_e/issues/46 @TitanVJ

**Comment by TitanVJ** [comment]
ok, ill pm storm and tell him to push to his branch?

**Comment by TitanVJ** [comment]
@modernNeo #46 is a branch of itself, should it just be a pr on it own? since this pr is for add mod specific commands and #46 deals with server admin stuff

**Comment by modernNeo** [comment]
@TitanVJ fair point, in that case I will approve this PR but tell them to submit it already, are they voting on what to say or something?

**Comment by TitanVJ** [comment]
pm'd storm about the issue. from wat i remember it was brought up once in council but i think they said watever u had from henry's old msg was fine. best to wait for storms response though cuz that was bit ago

**Comment by wnfldchen** [comment]
ok will wait on the response

**Pull request merged**